float64[360] range
---
int32 turnControl
